rashgaroth/vue-web3wallet-walletconnect
# add mechanism to perform personal-sign

# Description
Adding a mechanism to handle `personal sign` operation

# How to test
1. Connect the app with another dapp
2. perform a `personal sign` operation
3. Observe that the signature is valid
